.github/actions/whitesource-scan/action.yml

```diff
@@ -0,0 +1,140 @@
+name: WhiteSource Scan
+description: Used for performing WhiteSource scanning.
```

Comment:
This is the same as: https://github.com/SolaceDev/maas-build-actions/blob/master/.github/actions/whitesource-scan/action.yaml